### PR TITLE
Show different reply text based on selection / single reply

### PR DESCRIPTION
**Pull request by jpaidoussi**

### CODE REVIEW

**Comment by jpaidoussi** [comment]
@rudigiesler Please could you review this when you have a moment?

File: templates/partials/modal_reply.haml [hunk 1]
```diff
@@ -5,7 +5,10 @@
     - trans "Reply"
   .modal-body
     %p
```
Comment (rudigiesler) [inline templates/partials/modal_reply.haml]: Maybe we want to remove this `p`, else it's an unnecessary `p` nesting.

**Comment by rudigiesler** [comment]
Just the one small comment from me, looks good otherwise 😄

**Comment by rudigiesler** [comment]
:+1:

**Pull request merged**